apps/api/v2/src/modules/organizations/repositories/organizations-teams.repository.ts

```diff
@@ -55,4 +55,24 @@ export class OrganizationsTeamsRepository {
       take,
     });
   }
+
+  async findOrgTeams(organizationId: number) {
+    const teams = await this.dbRead.prisma.team.findMany({
+      where: {
+        OR: [
+          {
+            parentId: organizationId,
+          },
+          {
+            id: organizationId,
```

Comment:
Do we want to put organization itself and teams in the same box? From the business perspective I feel like I would exclude the `id: organizationId` rule because orgs and teams are perceived differently, and instead we could have a function `findOrg` if we needed one.